content/influxdb/cloud/query-data/parameterized-queries.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Use parameterized Flux queries
+description: >
+  Use parameterized queries to re-use queries and prevent injection attacks
+weight: 104
+menu:
+  influxdb_cloud:
+    name: Parameterized queries
+    parent: Query data
+influxdb/cloud/tags: [query, security]
+---
+
+Flux supports parameterized queries in InfluxDB Cloud.
+Parameterized queries make Flux queries more reusable, and can also be used to help prevent injection attacks
+
+Use parameterized queries to dynamically change values used in a query without having to re-write it.
+This feature allows users to define the values of variables in a separate field in a request payload.
+A parameterized query enables you to supply arguments which are then inserted into the Flux query for it to be executed.
+
+Use parameterized queries when executing Flux with untrusted user input;
+for example, in a web or IoT application.
+For more information on security and query parameterization,
+see the OWASP [SQL Injection Prevention Cheat Sheet][].
+While this guide is about SQL, it contains useful general advice.
+
+## Example
+
+To use a parameterized query, do the following:
+
+1. Create your Flux query.
+   ```js
+   from(bucket:params.mybucket)
+     |> range(start: -7d)
+     |> limit(n:2)","params":{"mybucket":"telegraf"}
+   ```
+   Here we pass in a bucket name as an argument to a parameterized Flux query.
+   The Flux engine will replace `params.mybucket` with the bucket name that we want to query.
```

Comment:
~~~suggestion
1. Create your Flux query.
   Use dot or bracket notation to reference parameters inside the `params`
   record to populate values at query time.
   The following example uses `params.mybucket` to define the bucket name.

    ```js
    from(bucket: params.mybucket)
      |> range(start: -7d)
      |> limit(n:2)
    ```
~~~